projects/plugins/jetpack/_inc/client/at-a-glance/backups.jsx

```diff
@@ -294,10 +287,12 @@ class DashBackups extends Component {
 						) }
 					</React.Fragment>
 				);
-			case 'active':
-				const message = containsBackupRealtime( planClass )
-					? __( 'We are backing up your site in real-time.', 'jetpack' )
-					: __( 'We are backing up your site daily.', 'jetpack' );
+			case 'active': {
+				/* Avoid ternary as code minification will break translation function. :( */
```

Comment:
Is this p2'd or described anywhere?

Reply:
I've not looked but it seems like I'm not the first one to have come across this
https://github.com/Automattic/jetpack/blob/a860290cfdf9978014e9ba2e34363cd9df6d4e88/projects/plugins/jetpack/_inc/client/components/support-card/index.jsx#L92

Reply:
It looks like it's a documented bug: https://github.com/Automattic/jetpack/tree/master/projects/js-packages/i18n-check-webpack-plugin#conditional-function-call-compaction